src/RayTracer/Public/js/render-3d.js

```diff
@@ -0,0 +1,161 @@
+const $ = require('jquery');
+
+// standard global variables
+let container, scene, camera, renderer, controls, stats;
+
+// custom global variables
+let targetList = [];
+let projector, mouse = { x: 0, y: 0 };
+	// FUNCTIONS 		
+
+	function rgbToHex(r, g, b) {
```

Comment:
Is this not the same function as in colour picker? If so, please import it rather than duplicating the code